tests/test_F0_output.py

```diff
@@ -0,0 +1,189 @@
+                    {"label": ["asset_a", "asset_b"], "optimizedAddCap": [1, 2]}
+                )
+            },
+        }
+        F0.plot_optimized_capacities(dict_scalar_capacities)
+        assert (
+            os.path.exists(
+                os.path.join(OUTPUT_PATH, "optimal_additional_capacities.png")
+            )
+            is True
+        )
+
+    def test_store_scalars_to_excel_two_tabs_dict(self):
+        dict_scalars_two_tabs_dict = {
+            "simulation_settings": {"path_output_folder": OUTPUT_PATH},
+            "kpi": {
+                "economic": pandas_Dataframe,
+                "technical": {"param1": 1, "param2": 2},
+            },
+        }
+        F0.store_scalars_to_excel(dict_scalars_two_tabs_dict)
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "scalars" + ".xlsx")) is True
+
+    def test_store_scalars_to_excel_two_tabs_no_dict(self):
+        dict_scalars_two_tabs = {
+            "simulation_settings": {"path_output_folder": OUTPUT_PATH},
+            "kpi": {"economic": pandas_Dataframe, "technical": pandas_Dataframe},
+        }
+
+        F0.store_scalars_to_excel(dict_scalars_two_tabs)
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "scalars" + ".xlsx")) is True
+
+    def test_store_each_bus_timeseries_to_excel_and_png_one_bus(self):
+        dict_timeseries_test_one_bus = {
+            "simulation_settings": {"path_output_folder": OUTPUT_PATH},
+            "optimizedFlows": {"a_bus": bus},
+        }
+
+        F0.store_timeseries_all_busses_to_excel(dict_timeseries_test_one_bus)
+        assert (
+            os.path.exists(os.path.join(OUTPUT_PATH, "timeseries_all_busses" + ".xlsx"))
+            is True
+        )
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "a_bus" + " flows.png")) is True
+
+    def test_store_each_bus_timeseries_to_excel_and_png_two_busses(self):
+        dict_timeseries_test_two_busses = {
+            "simulation_settings": {"path_output_folder": OUTPUT_PATH},
+            "optimizedFlows": {"a_bus": bus, "b_bus": bus},
+        }
+        F0.store_timeseries_all_busses_to_excel(dict_timeseries_test_two_busses)
+        assert (
+            os.path.exists(os.path.join(OUTPUT_PATH, "timeseries_all_busses" + ".xlsx"))
+            is True
+        )
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "a_bus" + " flows.png")) is True
+        assert os.path.exists(os.path.join(OUTPUT_PATH, "b_bus" + " flows.png")) is True
+
+    def test_store_dict_into_json(self):
+        file_name = "test_json_converter"
+        F0.store_as_json(json_test_dictionary, OUTPUT_PATH, file_name)
+        assert os.path.exists(os.path.join(OUTPUT_PATH, file_name + ".json")) is True
+
+    def teardown_class(self):
+        if os.path.exists(OUTPUT_PATH):
+            shutil.rmtree(OUTPUT_PATH, ignore_errors=True)
+
+
+class TestDictionaryToJsonConversion:
+    def setup_class(self):
+        os.mkdir(OUTPUT_PATH)
```

Comment:
I usually add a line to delete the folder if it exists (if the test crashed due to an error, the teardown method does not get executed and the folder might still be there). You could also simply use `self.teardown()` if this all what teardown does anyway :)

Reply:
Done, I think?